src/bullets/logger.py

```diff
@@ -33,6 +33,7 @@ def format(self, record):
 
 
 LOG_LEVEL = LogLevels.INFO
+SUPPRESS_LOGS = False
```

Comment:
I don't think the whole suppress logs logic is needed, if you change the log level, it will suppress all the levels below. For example, if you set the log level to warning, it suppresses Debug and Info.